modules/plugins/filetree/neo-tree/config.nix

```diff
@@ -21,7 +22,28 @@ in {
         setupModule = "neo-tree";
         inherit (cfg) setupOpts;
 
+        beforeAll =
```

Comment:
This should be after the lz.n spec is registered, because neo-tree can't be properly loaded before that point.

Reply:
just tested, trigger_load works in beforeAll

Reply:
It still doesn't make sense, come on

Reply:
idk why it matters but moved to pluginRC